Sources/DeepLearning/Random.swift

```diff
@@ -16,41 +16,32 @@
 import TensorFlow
 #endif
 
+private typealias Vector2 = (UInt32, UInt32)
```

Comment:
For clarity, I would suggest renaming these to `UInt32x2` or `UInt32x4`, respectively. When we merge from upstream next time, you can switch to `SIMD2<UInt32>` and `SIMD4<UInt32>`.